tests/behat/behat-features/src/FormContext.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Behat\Mink\Exception\ExpectationException;
+
+/**
+ * Defines application features from the specific context.
+ */
+class FormContext extends RawDrupalContext {
+
+  /**
+   * @Then :element_selector value is empty
```

Comment:
Empty checkbox? For that there are existing steps I think:

source dotenv/loader.sh; docker-compose  exec web ./vendor/bin/behat -c web/modules/drunomics/contentpool-client/tests/behat/behat.yml  -d check

outputs useful things like

default | [Then|*] /^the checkbox "(?P<checkbox>(?:[^"]|\\")*)" is (?:unchecked|not checked)$/
        | Checks, that checkbox with specified id|name|label|value is unchecked
        | Example: Then the "newsletter" checkbox should be unchecked
        | Example: Then the "newsletter" checkbox should not be checked
        | Example: And the "newsletter" checkbox is unchecked

Reply:
Checking if input field is empty (since teaser was removed).
Changed it to support:
* Example: Then Value of input field "[name='field_teaser_media[target_id]']" should be "empty"
* Example: Then Value of input field "[name='field_teaser_media[target_id]']" should be "1"

Add this to behat utils or ContentPoolContext?